tsconfig.app.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
+    "target": "ES2020",
```

Comment:
shouldn't we adhere to the setting in `tsconfig.node.json`? This also applies to 
````
    "lib": ["ES2023"],
    "module": "ESNext",
    "skipLibCheck": true,
````
Or was there a particular reason?